### DISCUSSION TITLE
此插件对PowerToys版本的要求

### DISCUSSION THREAD

**Discussion by themql** [N0I0C0K/PowerTranslator#6] [Q&A] [answered]
我的PowerToys版本为v0.59.1，无法加载该插件

现在电脑电池坏了，Windows版本无法更新，导致无法安装PowerToys更新版本

**Comment by N0I0C0K** [comment, marked as answer]
对，因为之前有一次大版本更新，整体更到了.net7，导致老版本无法安装。你可以尝试下载初代可用版本，那是基于老版本开发的。

针对老版本的问题，我其实更建议你更新到新版本，我不太清楚是什么原因导致的无法更新，你可以在Powertoys项目提交[issue](https://github.com/microsoft/PowerToys/issues)寻求帮助。

**Reply by themql** [reply]
感谢:)
